tensorrt_llm/_torch/auto_deploy/custom_ops/quant.py

```diff
@@ -218,4 +218,93 @@ def fp4_linear_fake(
     return torch.ops.aten.linear(input, weight_fp4.repeat(1, 2).to(input.dtype), bias)
 
 
-QUANT_OPS = [torch.ops.quant.fp8_linear, torch.ops.quant.fp4_linear]
+@torch.library.custom_op("quant::fp8_bmm", mutates_args=())
+@torch.compile(dynamic=True)
+def fp8_bmm(
+    input: torch.Tensor,
+    weight: torch.Tensor,
+    input_scale: torch.Tensor,
+    weight_scale: torch.Tensor,
```

Comment:
Can we stick to [`torch.bmm`](https://docs.pytorch.org/docs/stable/generated/torch.bmm.html) naming convention for these args:
```suggestion
    input: torch.Tensor,
    mat2: torch.Tensor,
    input_scale: torch.Tensor,
    mat2_scale: torch.Tensor,
```